element_interface/intanloader/rhsutilities.py

```diff
@@ -0,0 +1,1037 @@
+import sys, struct, math, os, time
+import numpy as np
+import matplotlib.pyplot as plt
+
+# Define plural function
+def plural(n):
+    """Utility function to optionally pluralize words based on the value of n."""
+
+    if n == 1:
+        return ""
+    else:
+        return "s"
+
+
+# Define read_qstring function
+def read_qstring(fid):
+    """Read Qt style QString.
+
+    The first 32-bit unsigned number indicates the length of the string (in bytes).
+    If this number equals 0xFFFFFFFF, the string is null.
+
+    Strings are stored as unicode.
+    """
+
+    (length,) = struct.unpack("<I", fid.read(4))
+    if length == int("ffffffff", 16):
+        return ""
+
+    if length > (os.fstat(fid.fileno()).st_size - fid.tell() + 1):
+        print(length)
+        raise Exception("Length too long.")
+
+    # convert length from bytes to 16-bit Unicode words
+    length = int(length / 2)
+
+    data = []
+    for i in range(0, length):
+        (c,) = struct.unpack("<H", fid.read(2))
+        data.append(c)
+
+    if sys.version_info >= (3, 0):
+        a = "".join([chr(c) for c in data])
+    else:
+        a = "".join([unichr(c) for c in data])
+
+    return a
+
+
+# Define read_header function
+def read_header(fid):
+    """Reads the Intan File Format header from the given file."""
+
+    # Check 'magic number' at beginning of file to make sure this is an Intan
+    # Technologies RHD2000 data file.
+    (magic_number,) = struct.unpack("<I", fid.read(4))
+    if magic_number != int("0xD69127AC", 16):
+        raise Exception("Unrecognized file type.")
+
+    header = {}
+    # Read version number.
+    version = {}
+    (version["major"], version["minor"]) = struct.unpack("<hh", fid.read(4))
+    header["version"] = version
+
+    print("")
+    print(
+        "Reading Intan Technologies RHS2000 Data File, Version {}.{}".format(
+            version["major"], version["minor"]
+        )
+    )
+    print("")
```

Comment:
```suggestion
```

Can we remove all these print statements?

Reply:
I'd suggest migrating all element `print` statements to datajoint-python-formatted logging

```python
import logging
logger = logging.getLogger('datajoint')
logger.info('existing statement')
```